apps/provider-scaffold/template/resources/providerid.go.template

```diff
@@ -0,0 +1,9 @@
+package resources
+
+func (k *mql{{ .CamelcaseProviderID }}) id() (string, error) {
+	return "{{ .ProviderID }}", nil
+}
+
+func (s *mql{{ .CamelcaseProviderID }}) field() (string, error) {
```

Comment:
Shouldn't this be the same as above? So, either `(s *mql...)` or `(k *mql...)`?

Reply:
definitly

Reply:
fixed